layouts/partials/sidebar/left.html

```diff
@@ -77,17 +77,19 @@ <h2 class="site-description">{{ .Site.Params.sidebar.subtitle }}</h2>
 
         <div class="menu-bottom-section">
             {{- $currentLanguageCode := .Language.Lang -}}
-            {{ with .Site.Home.AllTranslations }}
-                <li id="i18n-switch">  
-                    {{ partial "helper/icon" "language" }}
-                    <select name="language" onchange="window.location.href = this.selectedOptions[0].value">
-                        {{ range . }}
-                            <option value="{{ .Permalink }}" {{ if eq .Language.Lang $currentLanguageCode }}selected{{ end }}>{{ .Language.LanguageName }}</option>
-                        {{ end }}
-                    </select>
-                </li>
+            {{ if ( compare.Gt .Site.Home.AllTranslations 1 ) }}
```

Comment:
.Site.Home.AllTranslations` has type `Pages`. This fix works, but I think it's doing some kind of casting to compare a `Pages` to an `Int`.

I think it's better to avoid this kind of casting. We can use `.Site.Home.AllTranslations.Len` to get the length of the collection.